projects/ngrx.io/content/guide/store/testing.md

```diff
@@ -74,6 +74,12 @@ Usage:
 
 In this example, we mock the `getLoggedIn` selector by using `overrideSelector`, passing in the `getLoggedIn` selector with a default mocked return value of `false`.  In the second test, we use `setResult()` to update the mock selector to return `true`.
 
+<div class="alert is-helpful">
+
+**Note:** You should reset overridden selectors using `resetSelectors()` to prevent other tests from using the overridden selector value.  This is shown in the above example.
```

Comment:
```suggestion
**Note:** Reset overridden selectors using the `MockStore.resetSelectors()` method to prevent other tests from using the overridden selector value.  This is shown in the above example.
```